org.osgi.test.junit5/src/test/java/org/osgi/test/junit5/service/AbstractServiceExtensionTest.java

```diff
@@ -121,7 +121,7 @@ public void afterEach() throws Exception {
 		checkClass(testClass);
 		try {
 			return executor.schedule(() -> assertThatTest(testClass), delay, TimeUnit.MILLISECONDS)
-				.get(delay + 200000, TimeUnit.MILLISECONDS);
+				.get(200000L + delay, TimeUnit.MILLISECONDS);
```

Comment:
I don't object to this change but for my own personal education purposes I am curious as to the reasoning.

Reply:
The argument to `get` must be a long. The code had an int+int which makes an int which then must be widened to a long. If delay was, for example, Integer.MAX_VALUE then the int+int would overflow an int. By making the expression long+int, the compiler will first widen the int delay argument to long and then add to the constant long. So we never can overflow. I just happened to see this when looking at the code. I always try to use `L` for long constants and `D` for double constants to be explicit and avoid making the compiler infer what I mean.